### PR TITLE
docs(forms-web-app): add entry to GitHub PR template for env var changes

### PR DESCRIPTION
**Pull request by a6software**
## Ticket Number

UCD-

## Checklist

- [ ] Requires infrastructure changes
- [x] I have updated the documentation accordingly
- [ ] I have merged commits to avoid fixing things in this PR
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ UCD-
 ## Checklist
 <!-- Put an `x` in all the boxes that apply: -->
 - [ ] Requires infrastructure changes
+  - [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate helm chart
```
Comment (MrSimonEmms) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: Can this be formatted as lines above/below? It looks squiffy when viewed in GitHub

Also, "Helm" with capital H

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]

**Comment by MrSimonEmms** [comment]
I was also thinking about the wording of "I have merged commits to avoid fixing things in this PR".

How about replacing with "My commit history in this PR is linear"? I think the wording has been confusing people into merging everything into a single commit which is not always the most readable history

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (a6software) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Awww maaaan, it was deliberately nested under `Requires infrastructure changes`

No worries ;)

Comment (MrSimonEmms) [reply .github/PULL_REQUEST_TEMPLATE.md]: Aaaah, sorry. It didn't seem to put the bullet points when I viewed it

**Review by MrSimonEmms — APPROVED** [review]

**Pull request merged**